StRoot/StBFChain/BigFullChain.h

```diff
@@ -1734,6 +1737,10 @@ Bfc_st BFC[] = { // standard chains
                                                                              "sTGC Point maker",    kFALSE},
   {"FttQA","","fttChain","","StFttQAMaker","StFttQAMaker", "sTGC Raw hit QA maker",                 kFALSE},
 
+  // forward track maker
+  {"FwdTrack","","","","StFwdTrackMaker",
+  "XMLIO.so,genfit2.so,KiTrack.so,StarGeneratorUtil.so,libMathMore.so,StEventUtilities.so,StEpdUtil,StFwdTrackMaker", 
```

Comment:
I know there's a few other instances of including the ".so" suffix in libraries to load, but if it works without them, I'd recommend removing the suffix for a cleaner look (which is part of the point of this PR).